playbook/app/pb_kits/playbook/pb_card/card.html.erb

```diff
@@ -1,5 +1,22 @@
-<%= pb_content_tag(object.tag) do %>
-    <%= content.presence %>
+<% if object.draggable_item %>
+    <%= pb_rails("draggable/draggable_item", props:{drag_id: object.drag_id}) do %>
+        <%= pb_content_tag(object.tag) do %>
+            <% if object.draggable_item && object.drag_handle %>
```

Comment:
For this conditional, what happens if they want to make it draggable but NOT use the drag_handle? Will it still work? Did you test with drag_handle set to false?